### PR TITLE
Add option to round edges in SugiyamaEdgeRenderer

### PR DESCRIPTION
**Pull request by JulianBissekkou**
This PR introduces the option to round the edges of the Sugiyama algorithm.

### CODE REVIEW

**Comment by JulianBissekkou** [comment]
@nabil6391 Any time to take a look at this PR?

**Comment by nabil6391** [comment]
I have added a comment only in one place, other mostly LGTM. Thanks! @JulianBissekkou

**Comment by JulianBissekkou** [comment]
> I have added a comment only in one place, other mostly LGTM. Thanks! @JulianBissekkou

Did you forget to submit your review? I don't see the comment.

File: lib/layered/SugiyamaEdgeRenderer.dart [hunk 1]
```diff
@@ -56,8 +57,29 @@ class SugiyamaEdgeRenderer extends ArrowEdgeRenderer {
...
+
+          final x = bendPoints[i];
+          final y = bendPoints[i + 1];
+          final x2 = isLastPoint ? -1 : bendPoints[i + 2];
+          final y2 = isLastPoint ? -1 : bendPoints[i + 3];
+          if (x == x2 && y == y2) {
+            continue; // Skip when two consecutive points are identical
+            // because drawing a line between would be redundant in this case.
+          }
+          bendPointsWithoutDuplication.add(Offset(bendPoints[i], bendPoints[i + 1]));
```
[the remark this answers is not in the document]
Comment (JulianBissekkou) [reply lib/layered/SugiyamaEdgeRenderer.dart]: Done ✅

**Comment by JulianBissekkou** [comment]
Fixed the feedback. 👍🏽

**Comment by JulianBissekkou** [comment]
Thanks for merging. Let me know when you deployed a new version to pub :)

**Pull request merged**